### PR TITLE
fix(documentation-website): remove unnecessary media query

### LINKED ISSUE #608: remove @media screen and (width <= 786px)
## Describe the bug

We hhave a mobile first design. This media query contradicts that.

## Expected behavior

We don't have a smaller or equals query, but a bigger or equals than

### PR DESCRIPTION
**Pull request by itsmacr8**
# The Pull Request is ready

- [x] fixes #608 
- [ ] all actions are passing
- [x] only fixes a single issue

## Overview

remove unnecessary media query that were add for mobile devices. It is not needed because it is a mobile first design

## Documentation-Website

- [x] mobile view is usable
- [x] desktop view is usable
- [ ] no a-tags are used directly (NavLink, MailLink, ExternalLink instead)
- [ ] all new texts are added to the translation files (at least the english one)
- [ ] tests have been added (if required)
- [ ] shared code has been extracted in a different file

### CODE REVIEW

File: documentation-website/src/index.css [hunk 1]
```diff
@@ -559,21 +559,19 @@ html.dark-mode {
...
 
-@media screen and (width <= 786px) {
-  footer {
-    display: flex;
-    justify-content: center;
-    flex-wrap: wrap;
-    padding: 1rem;
-    gap: 1rem;
-  }
+footer {
```
Comment (Idrinth) [inline documentation-website/src/index.css:562]: the idea was to keep the functionality, this will likely break it.

**Review by Idrinth — CHANGES_REQUESTED** [review]

**Comment by Idrinth** [comment]
@coderabbitai review

**Comment by itsmacr8** [comment]
@Idrinth All checks passed. Is it okay to merge?

**Comment by Idrinth** [comment]
> @Idrinth All checks passed. Is it okay to merge?

yes, perfect!

**Pull request merged**